### PR TITLE
MUSARK-447

### PR DESCRIPTION
**Pull request by rituvesh**
Updated the picklist logic to call for more then one nodes or objects
Continue working and testing

### CODE REVIEW

File: src/containers/picklist/PickListContainer.js [hunk 1]
```diff
@@ -66,9 +66,9 @@ export default class PickListContainer extends React.Component {
   moveModal = (toId, toName) => {
     const moveFunction = this.isTypeNode() ? this.props.moveNode : this.props.moveObject
     const callback = this.isTypeNode() ? this.nodeCallback(toName) : this.objectCallback(toName)
-    this.state.itemsToMove.forEach((itemToMove => {
-      moveFunction(itemToMove.id, toId, this.props.user.id, callback)
-    }))
+    const arr = [];
```
Comment (jarlah) [inline src/containers/picklist/PickListContainer.js]: Rule #1 in functional programming, avoid mutability. Therefore do this instead. And save yourself a line:

```
const arr = this.state.itemsToMove.map(itemToMove => itemToMove.id)
```

also rename the arr variable to something more readable like toMove

**Review by jarlah — CHANGES_REQUESTED** [review]

File: src/reducers/move/index.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ export const moveNode = (nodeId, destination, doneBy, callback) => {
   const data = {
     doneBy,
     destination,
-    items: [nodeId]
+    items: Array.isArray(nodeId) ? nodeId : [nodeId]
```
Comment (jarlah) [inline src/reducers/move/index.js]: Simplify to:

```
items: [].concat(nodeId]
```

**Review by jarlah — CHANGES_REQUESTED** [review]

**Review by rituvesh — COMMENTED** [review]
Done!

**Review by steinaol — APPROVED** [review]
Seems like you have done what Jarl requested in earlier reviews.

**Review by steinaol — APPROVED** [review]

**Review by rituvesh — COMMENTED** [review]
Done

**Comment by steinaol** [comment]
I am not able to merge this yet. Some changes are requester or addressed.  Look at the PR

**Review by steinaol — APPROVED** [review]
ok

**Review by steinaol — APPROVED** [review]

**Pull request merged**